kura/org.eclipse.kura.asset.cloudlet.provider/OSGI-INF/AssetCloudletComponent.xml

```diff
@@ -1,16 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-    Copyright (c) 2016 Eurotech and/or its affiliates and others
+    Copyright (c) 2016, 2017 Eurotech and/or its affiliates and others
 
      All rights reserved. This program and the accompanying materials
      are made available under the terms of the Eclipse Public License v1.0
      which accompanies this distribution, and is available at
      http://www.eclipse.org/legal/epl-v10.html
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" 
-	activate="activate" 
-	deactivate="deactivate" 
-	name="org.eclipse.kura.asset.cloudlet">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" immediate="true" name="org.eclipse.kura.asset.cloudlet">
```

Comment:
I don't know why but Asset-V1 cloudlet instance should have been registered as soon as this service instance is tracked by a consumer. This is the usual scenario of a delayed component. As soon as a consumer requests for a delayed component instance, it gets instantiated. And since beginning, it was delayed and during testing and our presentations, it worked even though it was delayed.